tests/chocolatey-tests/choco-install.Tests.ps1

```diff
@@ -629,6 +637,15 @@ Describe "choco install" -Tag Chocolatey, InstallCommand {
             $XML.package.metadata.version | Should -Be "1.0.0"
         }
 
+        It "Does not output a warning message about side by side installs are deprecated" {
+            $Output.Lines | Should -Not -Contain "Installing the same package with multiple versions is deprecated and will be removed in v2.0.0." -Because $Output.String
+        }
+
+        It "Does not output a warning message that installed side by side package is deprecated" {

```

Comment:
Contrary to the prior comments, I don't think we need to version gate this, as the behaviour should not change.